docs/src/terminology.md

```diff
@@ -58,16 +58,29 @@ A gossip network connecting all [nodes](terminology.md#node) of a [cluster](term
 
 ## cooldown period
 
-Some number of epochs after stake has been deactivated while it progressively becomes available for withdrawal. During this period, the stake is considered to be "deactivating". More info about: [warmup and cooldown](implemented-proposals/staking-rewards.md#stake-warmup-cooldown-withdrawal)
+A period of some number of epochs after a user has signed a transaction to deactivate their stake. During this period, the stake progressively becomes available for withdrawal and is considered to be deactivating. As the stake [cools down](implemented-proposals/staking-rewards.md#stake-warmup-cooldown-withdrawal), it continues to earn rewards and be exposed to slashing. The rate of cooldown is configurable, and is [currently set to 25% on the Mainnet.](https://github.com/solana-labs/solana/blob/c7a2fbe7eba18772266f3b4b02ccfb66353ffa3e/programs/stake/src/config.rs#L15-L17)
 
 ## credit
 
 See [vote credit](terminology.md#vote-credit).
 
+## custodian
```

Comment:
I'm prefer we purge this term from the docs. Instead, we should exclusively use "lockup authority".